cmd/commands/init.go

```diff
@@ -0,0 +1,122 @@
+package commands
+
+import (
+	"io/ioutil"
+	"path"
+
+	"github.com/urfave/cli"
+
+	cmn "github.com/tendermint/go-common"
+)
+
+var InitCmd = cli.Command{
+	Name:      "init",
+	Usage:     "Initialize a basecoin blockchain",
+	ArgsUsage: "",
+	Action: func(c *cli.Context) error {
+		return cmdInit(c)
+	},
+	Flags: []cli.Flag{
+		ChainIDFlag,
+	},
+}
+
+func cmdInit(c *cli.Context) error {
+	rootDir := BasecoinRoot("")
+
+	cmn.EnsureDir(rootDir, 0777)
+
+	// initalize basecoin
+	genesisFile := path.Join(rootDir, "genesis.json")
+	privValFile := path.Join(rootDir, "priv_validator.json")
+	key1File := path.Join(rootDir, "key.json")
+	key2File := path.Join(rootDir, "key2.json")
+
+	if err := ioutil.WriteFile(genesisFile, []byte(genesisJSON), 0644); err != nil {
+		return err
+	}
+	if err := ioutil.WriteFile(privValFile, []byte(privValJSON), 0400); err != nil {
+		return err
+	}
+	if err := ioutil.WriteFile(key1File, []byte(key1JSON), 0400); err != nil {
+		return err
+	}
+	if err := ioutil.WriteFile(key2File, []byte(key2JSON), 0400); err != nil {
+		return err
+	}
+
+	log.Notice("Initialized Basecoin", "genesis", genesisFile, "key", key1File)
+
+	return nil
+}
+
+const privValJSON = `{
```

Comment:
It's just simpler than needing to use a file. We should probably have no defaults and generate new keys/genesis on init but then we can't have a deterministic tutorial.